README.md

```diff
@@ -3,15 +3,16 @@
 This Web3.js plugin adds support for the following wallet-related RPC methods:
 
 - [wallet_addEthereumChain (EIP-3085)](https://eips.ethereum.org/EIPS/eip-3085)
```

Comment:
Since you link the EIPs in the descriptions below, I think these links should point to the anchors for the respective method descriptions.

Reply:
done per function in the examples below, here is the link to the full EIP